### ISSUE TITLE
Help needed with Classic Theme for Windows 10

### ISSUE THREAD

**Issue by malvinas2** [nptr/msstyleEditor#17]
Hello, I am aware that the issue-section of github is normally not used to discuss user problems, but I could not found any other forum. 
I tried to make some changes on a Classic Theme for Windows 10, which I uploaded to [http://www.share-online.biz/dl/Q3IPJIIP19EO](http://www.share-online.biz/dl/Q3IPJIIP19EO) (ca. 12 MB)

1) On the left side in the Explorer the font style appears blurred, unclear, while on the right side everything looks normal. 
How can I change this font to get rid of the blur?

2) When selecting, marking files and directories on the right side, the font color should change from black to white. How do I achieve this? "Textcolor" in ExplorerListView-->Listitem doesn't seem to work. 

3) Is it possible to change the font size of the main menu ("File", "Edit" etc.) ? 

Thank you very much, malvinas2

(the theme where I already did some changes is ".... Fall Creators 1709 / Show Commandbar / Windows Classic LightCaption.theme")

[https://i.imgur.com/j6nOOMr.png](https://i.imgur.com/j6nOOMr.png)

**Comment by nptr** [comment]
Hello, i am afraid you are experiencing some limitations of my program.

1.) The part responsible for the left side of the explorer should be `Explorer::TreeView`. Now in your screenshot i can't see any blur, just that the font is larger. Problem is, font support poor atm, especially in the latest release (1.3.0.0), see Issue #15. Your best chance to change the font (including size and style) is:
1. Use version 1.2.3.0 to see (and change if required) the FONT (ID) properties
2. Use a resource hacking tool to edit the actual font entries behind the IDs.

The font entries are typically located in the .msstyles file in the `String Table` resource. For more infos, see Issue #15 as well.

2.) After a bit of experimenting i found that various props in `ItemView` had an effect. You want to look into them.

3.) The main menu of the explorer is defined in the `Toolbar` class. The main menu of many other programs is defined in the `Menu` class. I haven't seen a FONT (ID) property there and adding it is currently unsupported, so maybe you are out of luck here.

I still hope you can realize most of your plans despite the difficulties.
PS: i am working on improving font support, but it takes time :/

-- nptr

**Comment by malvinas2** [comment]
Thank you very much for your help. 
I managed to change all fonts to cleartype style. I have also tried the ItemView property, but finally decided to use a brighter color for selected files. In case you are interested in a classic theme, you can find it at [http://www.share-online.biz/dl/ZYL4YRKPP8
](http://www.share-online.biz/dl/ZYL4YRKPP8
)
Do you know by any chance which property controls the text color of the marked item shown at [https://imgur.com/m6pxe4N](https://imgur.com/m6pxe4N) ? (I don't expect you to do a big research, just in case you know it).

**Comment by nptr** [comment]
Assuming from your image you understand german, so i might link this:
https://www.deskmodder.de/wiki/index.php/Vista_Style_Builder:_Explorer

I think the english original is [this ](http://virtualcustoms.net/showthread.php/28810-Complete-Guide-to-Theme-Building)one, but the images seem to be gone or just for logged-in users.

Its for the VSB/WSB and not my tool, but the class names match!
TL;DR: This bar is called "Command Bar" and the `CommandModule` class is what you are looking :)
